content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Application server configuration
+linkTitle: Server config
+description: Learn how to define agent paths for Java application servers
+weight: 15
+cSpell:ignore: asadmin binsetenv binstart Glassfish Payara setenv
+---
+
+When instrumenting an application running on an application server, you've to
+define the path to the OpenTelemetry Java agent. The way to define the path
+differs from server to server.
+
+## JBoss EAP / WildFly
```

Comment:
I think it would be nice to start with the generic option: `JAVA_TOOLS_OPTIONS`
https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html

This should help with many situations that aren't explicitly defined here.

Reply:
Added!